tests/build_disk_index.cpp

```diff
@@ -70,24 +74,23 @@ int main(int argc, char **argv)
                            "Storage type of Labels <uint/ushort>, default value is uint which "
                            "will consume memory 4 bytes per filter");
 
-        po::variables_map vm;
-        po::store(po::parse_command_line(argc, argv, desc), vm);
-        if (vm.count("help"))
-        {
-            std::cout << desc;
-            return 0;
-        }
-        po::notify(vm);
-        if (vm["append_reorder_data"].as<bool>())
-            append_reorder_data = true;
-        if (vm["use_opq"].as<bool>())
-            use_opq = true;
-    }
-    catch (const std::exception &ex)
-    {
-        std::cerr << ex.what() << '\n';
-        return -1;
+
+    po::variables_map vm;
+    po::store(po::parse_command_line(argc, argv, desc), vm);
+    if (vm.count("help")) {
+      std::cout << desc;
+      return 0;
     }
+    po::notify(vm);
+    if (vm["append_reorder_data"].as<bool>())
+      append_reorder_data = true;
+    if (vm["use_opq"].as<bool>())
+      use_opq = true;
+  } catch (const std::exception& ex) {
+    std::cerr << ex.what() << '\n';
+    return -1;
+  }
+  codebook_prefix = codebook_prefix == "0"? "" : codebook_prefix;
```

Comment:
Why is the default value 0 here?

Reply:
it's easier for the loading script in PowerShell to pass in "0" rather than empty. maybe I will rewrite it to be easier to understand.